packages/component/src/Utils/textFormatToContentType.js

```diff
@@ -0,0 +1,12 @@
+export default function (textFormat) {
+  switch (textFormat) {
+    case 'plain':
+      return 'text/plain';
+
+    case 'xml':
+      return 'text/xml';
+
+    default:
```

Comment:
not sure if this is the right default.

Reply:
Yes, this is the right default, same as the old day. 😉

If the user doesn't provide a Markdown implementation (not using full bundle), we will fallback to `text/plain`.